build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -45,11 +45,14 @@
       <Version>7.2.1</Version>
     </PackageReference>
     <PackageReference Include="NuGet.CommandLine.XPlat">
-      <Version>4.0.0-rc2</Version>
+      <Version>4.0.0-rc3</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Build.Utilities.Core">
       <Version>$(CLI_MSBuild_Version)</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.Build.Framework">
```

Comment:
ok. Since the build is broken anyways, can you add a comment with a pointer to the associated NuGet/Home issue so we know to remove this later?